sentry_sdk/tracing.py

```diff
@@ -123,7 +132,31 @@ def __repr__(self):
             )
         )
 
+    def __enter__(self):
+        # type: () -> Span
+        hub = self.hub or sentry_sdk.Hub.current
+
+        _, scope = hub._stack[-1]
+        old_span = scope.span
+        scope.span = self
+        self._context_manager_state = (hub, scope, old_span)
+        return self
+
+    def __exit__(self, ty, value, tb):
```

Comment:
I assume `value` is some kind of error here? Or exception parameter? Thats why it fails the transaction?

Reply:
`value` is an exception object while `ty` is its type and `tb` a traceback object.